model/registry/user.yaml

```diff
@@ -33,16 +33,11 @@ groups:
         note: >
           It can be a random guid or a hash of the user's IP address. This is different from `user.hash` which is a hash of a known `user.id` or `user.name`.
         examples: ['QdH5CAWJgqVT4rOr0qtumf']
-      - id: authentication
-        prefix: user.authentication
-        type: attribute_group
-        brief: "Describes information about the user authentication."
-        attributes:
-          - id: id
-            type: string
-            brief: "Unique identifier of an authenticated user in the system."
-            examples: ['S-1-5-21-202424912787-2692429404-2351956786-1000']
-            stability: experimental
+      - id: authentication.id
+        type: string
+        brief: "Unique identifier of an authenticated user in the system."
+        examples: ['S-1-5-21-202424912787-2692429404-2351956786-1000']
+        stability: experimental
```

Comment:
@trisch-me i tried to use nested `attribute_group` to represent new sub-namespace `authentication`. `make` command didn't like it and failed to update markdown accordingly. any suggestion?

Reply:
hey, I'm not sure what have you tried to do? The way I see it in final yaml is good one. You don't need to extract 1 subgroup in it's own attribute_group